src/domain/index.ts

```diff
@@ -68,10 +74,14 @@ export function init(repositories: Repositories, appConfig: AppConfig): DomainSe
     /**
      * @todo find a way how to resolve circular dependency
      */
+    noteSettings: noteSettingsService,
   });
   const aiService = new AIService(repositories.aiRepository);
 
+  const fileUploaderService = new FileUploaderService(repositories.objectStorageRepository, repositories.fileRepository, noteSettingsService);
```

Comment:
do not pass one service to another, Use shared interface instead

Reply:
for now, removed it